packages/frontend/core/src/commands/affine-creation.tsx

```diff
@@ -4,16 +4,19 @@ import type { createStore } from 'jotai';
 
 import { openCreateWorkspaceModalAtom } from '../atoms';
 import type { usePageHelper } from '../components/blocksuite/block-suite-page-list/utils';
+import { mixpanel } from '../mixpanel';
 import { registerAffineCommand } from './registry';
 
 export function registerAffineCreationCommands({
   store,
   pageHelper,
   t,
+  moduleName,
 }: {
   t: ReturnType<typeof useI18n>;
   store: ReturnType<typeof createStore>;
   pageHelper: ReturnType<typeof usePageHelper>;
+  moduleName: string;
```

Comment:
here are several issues

moduleName doesn't seem to update responsively
registering so many commands has a significant impact on performance, so it is not recommended to add dependencies.

I think we can just delete this and I'll implement it in another way later.

Reply:
Should I delete this event tracking directly or remove the "page, segment, module" attributes?